frontend/src/routes/(frontend)/about/index.tsx

```diff
@@ -27,9 +25,7 @@ type AboutInfo = {
 	}
 }
 
-export const aboutInfoLoader = loader$<Promise<AboutInfo>>(async ({ resolveValue, request, platform, html }) => {
-	throw html(404, getNotFoundHtml())
```

Comment:
here and for the settings page I thought of having an env and based on that 404ing or not

but I figured that it'd be a bit overengineered and not really that needed/useful (we can check social.eng first before synching cloudflare.social)

(also AFAIK we don't even provide a way to access those pages besides directly going to them via their url, so people probably won't try to access them)